### DISCUSSION TITLE
v46.0

### DISCUSSION THREAD

**Discussion by westnordost** [streetcomplete/StreetComplete#4332] [Release Discussions]
<h3>New Quests</h3>
  <p>Asked for campsites (#4213), by @mnalis:</p>
  <ul>
  <li><i>Who may camp here?</i></li>
  <li><i>Are there showers here?</i></li>
  <li><i>Is drinking water available here?</i></li>
  <li><i>Is there any power supply here?</i></li>
  </ul>
  <h3>Disabled Quests</h3>
  <p><i>Orchard produce</i>, <i>Building levels</i>, <i>Sidewalks</i> and <i>Lit roads</i> are now
  disabled by default because either they are replaced by designated overlays or they overwhelm
  users (#4264,...).<br/>
  You can enable them again in the settings at any time.
  </p>
  <h3>Enhancements on quests</h3>
  <ul>
  <li>Road surface: Ask for confirmation when selection contradicts a previous info (#4105, #662 #4063), by @matkoniecz</li>
  <li>Opening hours: Months and weekdays are now (also) shown in the local language</li>
  <li>All bus stop quests: Ask for any public transport stops, e.g. subway, train, ferry, ...(#3374)</li>
  <li>Road width: Also ask for the max width of chokers and chicanes (#3258, #4323)</li>
  <li>Recycling: Support PET as selectable choice (#2885)</li>
  <li>Tactile paving quests now enabled in Portugal (#4325)</li>
  <li>Internet access: Do not ask for backcountry campsites</li>
  <li>Toilet availability: Ask for campsites, too (#4330), by @mnalis</li>
  <li>Steps incline: Fix display issue on Android 5 (#4317)</li>
  <li>Buildings: Remove "show more" button (#3387, #4215), by @matkoniecz</li>
  </ul>
  <h3>Other enhancements and fixes</h3>
  <ul>
  <li>Fix editing tunnels in overlays (#4269), thanks @matkoniecz</li>
  <li>Fix rare error when uploading too long opening hours (#3471)</li>
  <li>Fix rare faulty links in notes (#4312, #4309)</li>
  <li>Improved map style clarity, especially in regard to overlays (#4291, #4310, ...)</li>
  <li>Performance improvement on uploading (#4297), by @Helium314</li>
  <li>Android 13: Only ask about notification permission when about to show one (#4294)</li>
  <li>More minor enhancements and fixes (#4158, #4226, #4307, #4302, #4271...), thanks @Helium314, @peternewman</li>
  </ul>

<hr /><em>This discussion was created from the release <a href='https://github.com/streetcomplete/StreetComplete/releases/tag/v46.0'>v46.0</a>.</em>

**Comment by nichu42** [comment]
Are updates no longer made available via F-Droid? The latest version that I could find there is 40.2 from February.

**Reply by FloEdelmann** [reply]
* See #4379
* Caused by #4289

**Reply by nichu42** [reply]
Thanks a lot. I hope there will be a solution soon. Or at least an alternative repository that can be used with F-Droid.

**Reply by mnalis** [reply]
Now that this has been worked around, newest StreetComplete `v47.0` is available on f-droid
